src/index.html

```diff
@@ -7,8 +7,91 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=""
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        src="images/logo.png"
+        alt="Moyo"
+        class="logo"
+      >
```

Comment:
did you check all points from [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)? 
also recheck 
![image](https://user-images.githubusercontent.com/64437769/142406080-27faee7e-6163-4883-9191-0e57d868cd15.png)